js/library.js

```diff
@@ -0,0 +1,81 @@
+const PEOPLE = {
+  name: 'Name',
+  birth_year: 'Birth year',
+  gender: 'Gender',
+  height: 'Height',
+  eye_color: 'Eye color',
+};
+
+const PLANETS = {
+  name: 'Name',
+  rotation_period: 'Rotation period',
+  diameter: 'Diameter',
+  climate: 'Climate',
+  gravity: 'Gravity',
+};
+
+const FILMS = {
+  title: 'Title',
+  director: 'Director',
+  producer: 'Producer',
+  release_date: 'Release date',
+};
+
+const SPECIES = {
+  name: 'Name',
+  classification: 'Classification',
+  average_height: 'Average height',
+  skin_colors: 'Skin colors',
+  language: 'Language',
+};
+
+const VEHICLES = {
+  name: 'Name',
+  model: 'Model',
+  manufacturer: 'Manufacturer',
+  cost_in_credits: 'Cost in credits',
+  crew: 'Crew',
+};
+
+const STARSHIPS = {
+  name: 'Name',
+  model: 'Model',
+  manufacturer: 'Manufacturer',
+  cost_in_credits: 'Cost in credits',
+  passengers: 'Passengers',
+};
+
+const CATEGORIESKEYS = {
+  people: 'people',
+  planets: 'planets',
+  films: 'films',
+  species: 'species',
+  vehicles: 'vehicles',
+  starships: 'starships',
+};
+
+const MENUSTYLING = {
```

Comment:
rename you constants according to what we discussed

Reply:
```suggestion
const LEFT_MENU_CLASS_NAME = {
```
Renamed all constants